pypika/tests/test_inserts.py

```diff
@@ -160,6 +160,14 @@ def test_non_insert_on_conflict_do_update(self):
             query = PostgreSQLQuery.update(self.table_abc).set('foo', 'bar').on_conflict(
                   'id').do_update(['name'], ["m"])
 
+    def test_insert_on_fieldless_conflict_do_nothing(self):
+        query = PostgreSQLQuery.into(self.table_abc).insert(1).on_conflict(None).do_nothing()
```

Comment:
Maybe it would be more obvious to not pass an argument to `on_conflict` instead of an explicit `None`? What do you think?